crates/typed-store-derive/src/lib.rs

```diff
@@ -564,10 +564,20 @@ pub fn derive_dbmap_utils_general(input: TokenStream) -> TokenStream {
                 }
             }
 
+            fn cf_name_to_table_name(cf_name: &str) -> eyre::Result<&'static str> {
```

Comment:
I'm not sure how to read this yet. Does it ensure that `live_owned_object_markers` reuse the cf of the existing `owned_object_transaction_locks` 's cf?

Do you think it worth adding a test to verify the upgrade flow (not sure how to do that though, or it may require a lot of effort)? The test added in this PR has the flag upgrade, but I think both new tables exist at the beginning of the test.

Or this can also be manually verified.

Reply:
Yes, both tables exist in the test, but I think other than that the test covers what we want? We add tables often enough I don't think its a cause for concern on its own?